src/ProjectFileTools/extension.vsixmanifest

```diff
@@ -13,6 +13,7 @@
   <Assets>
     <Asset Type="Microsoft.VisualStudio.VsPackage" Path="|%CurrentProject%;PkgdefProjectOutputGroup|" />
     <Asset Type="Microsoft.VisualStudio.MefComponent" Path="|ProjectFileTools.NuGetSearch|" />
+    <Asset Type="Microsoft.VisualStudio.MefComponent" Path="|ProjectFileTools.MSBuild|" />
```

Comment:
@mlorbetske are you talking about the Microsoft.Language.Xml.dll? If so, that should not be part of the <Assets>. That tag is used for when you own the loading of a dll, and is not intended for loading 3rd party dll's. All that is needed is for that .dll to be included in the vsix in one way or another, which Adrian did via the project file.